libfwupdplugin/fu-hid-device.h

```diff
@@ -37,10 +37,13 @@ typedef enum {
 	FU_HID_DEVICE_FLAG_ALLOW_TRUNC		= 1 << 0,
 	FU_HID_DEVICE_FLAG_IS_FEATURE		= 1 << 1,
 	FU_HID_DEVICE_FLAG_RETRY_FAILURE	= 1 << 2,
+	FU_HID_DEVICE_FLAG_NO_BIND_ON_CLOSE	= 1 << 3,
```

Comment:
why is this introduced in this commit?  (It's removed later)

Reply:
Oops, rebase failure -- sorry. Fixed now.